src/node_process_methods.cc

```diff
@@ -250,13 +251,44 @@ static void GetActiveRequests(const FunctionCallbackInfo<Value>& args) {
     AsyncWrap* w = req_wrap->GetAsyncWrap();
     if (w->persistent().IsEmpty())
       continue;
+
     request_v.emplace_back(w->GetOwner());
   }
 
   args.GetReturnValue().Set(
       Array::New(env->isolate(), request_v.data(), request_v.size()));
 }
 
+static void GetActiveRequests2(const FunctionCallbackInfo<Value>& args) {
```

Comment:
A more different name than just adding a number would be nice to be more descriptive about the actual difference.

Reply:
@Qard updated the names :)